### ISSUE TITLE
pip install itk fails

### ISSUE THREAD

**Issue by kirbyju** [InsightSoftwareConsortium/ITK#4451]
### Description

When running !pip install itk from a Jupyter Notebook on a Mac with Sonoma 14.3 and Python 3.12.1 I get this error:

```
Collecting itk
  Using cached itk-5.0.1.tar.gz (13 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [4 lines of output]
      scikit-build is required to build from source.
      Please run:
      
        python -m pip install scikit-build
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

### Steps to Reproduce

1. Just run "pip install itk".

### Expected behavior

Install itk

### Actual behavior
Error message mentioned above

### Reproducibility

100% of the time I try it

### Versions
5.3.0

### Environment

Mac OS Sonoma 14.3 and Python 3.12

### Additional Information

n/a

**Comment by dzenanz** [comment]
Wheels for 3.12 are not yet available for a stable version, see #4449. Please use an older Python version, or an ITK release candidate wheels.

**Comment by kirbyju** [comment]
Thanks, and sorry for not noticing the other issue was related.  I was able to get itk 3.5.0 to install after downgrading to python 3.11.
